lib/block-supports/duotone.php

```diff
@@ -354,11 +354,14 @@ function ( $selector ) use ( $duotone_id ) {
 	return $content . $duotone;
 }
 
-// Register the block support.
-WP_Block_Supports::get_instance()->register(
-	'duotone',
-	array(
-		'register_attribute' => 'gutenberg_register_duotone_support',
-	)
-);
-add_filter( 'render_block', 'gutenberg_render_duotone_support', 10, 2 );
+// This can be removed when plugin support requires WordPress 5.8.0+.
```

Comment:
This basically uses the wp-code if it is present. It makes future changes in the plugin impossible to do because the core version is going to be used. Could we remove the core filters instead if they are present?

Reply:
Makes sense. I'll update the code.

This is a pattern I've seen before in Gutenberg core and the only reason I've used it here.

Reply:
Done ✅ 

P.S. I pushed a similar fix for Group Block [layout support](https://github.com/WordPress/gutenberg/pull/33045). Do you want me to update logic there as well to use `remove_filter`?